### PR TITLE
Prop types and conflicting dependencies

### PR DESCRIPTION
**Pull request by rkrupinski**
Here's the most important change:

```diff
-    "@devexpress/dx-react-core": "1.0.0-beta.1",
-    "@devexpress/dx-react-grid": "1.0.0-beta.1",
-    "@devexpress/dx-react-grid-material-ui": "1.0.0-beta.1",
+    "@devexpress/dx-react-core": "1.0.0-beta.2",
+    "@devexpress/dx-react-grid": "1.0.0-beta.2",
+    "@devexpress/dx-react-grid-material-ui": "1.0.0-beta.2",
```

### CODE REVIEW

File: frontend/src/components/common/list/paginatedList.js [hunk 1]
```diff
@@ -111,70 +113,75 @@ class PaginatedList extends Component {
...
+            <Template
+              name="tableViewRow"
+              // use custom template only for table data rows
+              predicate={({ tableRow }) => tableRow.type === 'data'}
+            >
+              {params => (
+                <TemplateConnector>
+                  {(getters, actions) => (
+                    <TemplateRenderer
+                    // custom template
```
Comment (momimomo) [inline frontend/src/components/common/list/paginatedList.js]: comments should be left?

File: frontend/src/components/forms/textFieldForm.js [hunk 2]
```diff
@@ -44,7 +44,6 @@ function TextFieldForm(props) {
           infoText={infoText}
           warn={warn ? warning : null}
           formControlStyle={formControlStyle}
-          commentFormControlStyle={commentFormControlStyle}
```
Comment (momimomo) [inline frontend/src/components/forms/textFieldForm.js]: this is used to override styles in DS/R partner info, is there other solution implemented?

File: frontend/src/helpers/formHelper.js [hunk 3]
```diff
@@ -430,6 +429,7 @@ export const renderText = ({
   date,
   meta,
   multiline,
+  inputProps,
```
Comment (momimomo) [inline frontend/src/helpers/formHelper.js]: aren't inputProps inside InputProps?

File: frontend/src/components/common/list/paginatedList.js (hunk 1 above)
Comment (rkrupinski) [reply frontend/src/components/common/list/paginatedList.js]: That's actually a moved chunk of code. Here's a commit that introduced it:

https://github.com/unicef/un-partner-portal/commit/6f851b4bb21b0c0d1c40c2d2af1e127c0f5ea9a5#diff-8da2486a6e42cd8cc6803deb5acb6a05

cc @kenzo23312

File: frontend/src/components/forms/textFieldForm.js (hunk 2 above)
Comment (rkrupinski) [reply frontend/src/components/forms/textFieldForm.js]: The `commentFormControlStyle ` prop is still used here:

https://github.com/unicef/un-partner-portal/blob/fa5d6545447e7b5cc82f4d55f6d0af4f27c218e3/frontend/src/components/forms/textFieldForm.js#L27

The removed prop was passed to `renderTextField` and further along to `TextField` none of which expected it.

File: frontend/src/helpers/formHelper.js (hunk 3 above)
Comment (rkrupinski) [reply frontend/src/helpers/formHelper.js]: That's not the point here. `renderText` does neither use `InputProps` nor `inputProps`. However, previously only `InputProps` was destructured (removed from `...other`) thus `inputProps` was included in `Typography` props (see https://github.com/unicef/un-partner-portal/blob/fa5d6545447e7b5cc82f4d55f6d0af4f27c218e3/frontend/src/helpers/formHelper.js#L481), which caused an error. Line `432` effectively removes `inputProps` from `...other`.

**Pull request merged**